rfcs/text/pe/pe.yml

```diff
@@ -0,0 +1,193 @@
+---
+- name: pe
+  type: group
+  fields:
+    - name: main_icon
+      level: extended
+      type: object
+      description: >
+        Hashes of embedded program icon.
+      fields:
+        - name: dhash
+          level: extended
+          type: keyword
+          description: >
+            Difference Hash (dhash) for a given PE file.
```

Comment:
Can the description mention that this is about image-based visual differential hashing for the icon itself (not the executable code)?